source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/geo/Geometry2GML.java

```diff
@@ -124,16 +130,22 @@ private static AbstractRingPropertyType setCoordinates(final ObjectFactory facto
 
   /**
    * Set coordinates from wkt line string to gml LineStringType
```

Comment:
Might as well remove this whole comment, it doesn't really make sense... (or update it to something that does ;))

Reply:
Oh, frankly I meant to remove the fluff in the other method instead of this one :sweat:

Reply:
--- removed the parameter fluff in the other method and left this one untouched